pkg/workloads/k8s.go

```diff
@@ -226,6 +239,57 @@ func UserPodContainers(api spec.API) ([]kcore.Container, []kcore.Volume) {
 	return containers, volumes
 }
 
+func RealtimeProxyContainer(api spec.API) (kcore.Container, kcore.Volume) {
+	return kcore.Container{
+		Name:            _proxyContainerName,
+		Image:           config.ClusterConfig.ImageProxy,
+		ImagePullPolicy: kcore.PullAlways,
+		Args: []string{
+			"-port",
+			consts.ProxyListeningPortStr,
+			"-admin-port",
+			consts.ProxyAdminPortStr,
+			"-metrics-port",
+			consts.MetricsPortStr,
+			"-user-port",
+			s.Int32(*api.Pod.Port),
+			"-max-concurrency",
+			s.Int32(int32(api.Autoscaling.MaxConcurrency)),
+			"-max-queue-length",
+			s.Int32(int32(api.Autoscaling.MaxQueueLength)),
+			"-cluster-config",
+			consts.DefaultInClusterConfigPath,
```

Comment:
the flags can have a double dash at the beginning (e.g. `--port`, `--cluster-config`, etc)